src/poker/game/ActionHandlerTests.java

```diff
@@ -0,0 +1,14 @@
+package poker.game;
+
+import poker.Deck;
+import poker.EquityEvaluator;
+
+public class ActionHandlerTests extends ActionHandler{
+    public ActionHandlerTests(GameState gameState, HandsAndBoard result, Deck deck, Player player1, Player player2, EquityEvaluator equityEvaluator) {
+        super(gameState, result, deck, player1, player2, equityEvaluator);
+    }
+    @Override
+    public double scannerNextInt(){
+        return 10;
+    }
```

Comment:
byniu nie no poszedles na za duza latwizne to powinno sie moc podawac z listy i zrobic jakies scenatiusze ze raz betuje 10 a raz 20 i 40 itp a nie na sztywno zanwsze 10